### PR TITLE
add new nbiot parameters

### PR DESCRIPTION
**Pull request by cleresd**

### CODE REVIEW

File: src/analog/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -1965,6 +2022,42 @@ class CommandBinaryBuffer extends BinaryBuffer {
...
+        const bands: Array<number> = [];
+
+        for ( let index = 0; index < count; index++ ) {
+            bands.push(this.getUint8());
+        }
+
+        return {
+            count,
+            bands
+        };
```
Comment (DeadByElpy) [inline src/analog/CommandBinaryBuffer.ts:2046]:
```suggestion
        return {count, bands};
```

File: src/analog/commands/downlink/SetParameter.ts [hunk 2]
```diff
@@ -350,6 +350,21 @@ const examples: TCommandExampleList = [
...
+        name: 'set nbiot bands',
+        parameters: {
+            id: deviceParameters.NBIOT_BANDS,
+            data: {
+                count: 3,
+                bands: [
+                    3,
+                    8,
+                    20
+                ]
```
Comment (DeadByElpy) [inline src/analog/commands/downlink/SetParameter.ts:364]:
```suggestion
                bands: [3, 8, 20]
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: src/analog/commands/uplink/GetParameterResponse.ts [hunk 3]
```diff
@@ -214,6 +214,36 @@ const examples: TCommandExampleList = [
...
+            body: '33 10 42 43 36 36 30 4b 47 4c 41 41 52 30 31 41 30 35'
+        }
+    },
+    {
+        name: 'nbiot bands',
+        parameters: {
+            id: deviceParameters.NBIOT_BANDS,
+            data: {
+                count: 2,
+                bands: [
```
Comment (DeadByElpy) [inline src/analog/commands/uplink/GetParameterResponse.ts:237]: https://github.com/jooby-dev/jooby-codec/pull/377#discussion_r1575050566

File: src/analog/CommandBinaryBuffer.ts (hunk 1 above)
Comment (cleresd) [reply src/analog/CommandBinaryBuffer.ts]: ++

File: src/analog/commands/downlink/SetParameter.ts (hunk 2 above)
Comment (cleresd) [reply src/analog/commands/downlink/SetParameter.ts]: ++

File: src/analog/commands/uplink/GetParameterResponse.ts (hunk 3 above)
Comment (cleresd) [reply src/analog/commands/uplink/GetParameterResponse.ts]: ++

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**